conversion/conversion.go

```diff
@@ -390,6 +391,7 @@ func getSeekable(f *os.File) (*os.File, int64, error) {
 		return f, n, err
 	}
 	internal.VerbosePrintln("Creating a tmp file with a copy of stdin because stdin is not seekable.")
```

Comment:
Should we delete this line?

Reply:
Yeah, we need to keep these for compatibility with the `--verbose` mode that is present in the global command line mode. Once global mode is deprecated, we can remove these.